ts/src/profiler.ts

```diff
@@ -126,6 +137,48 @@ async function profileBytes(p: perftools.profiles.IProfile): Promise<string> {
 }
 
 /**
+ * Error which also indicates how long backoff should be in response to the
```

Comment:
Always indicates?

Reply:
I've changed how I use this error and updated the comment.